3scale/3scale-operator
# disable replicas reconciliation when annotation is present

This resolves the https://issues.redhat.com/browse/THREESCALE-8374

Verification
- deploy 3scale
- add annotation to the apiManager 
1. "apps.3scale.net/disable-apicast-production-replica-reconciler": "true"
2. confirm that the apicast production replicas aren't synced anymore then set the above annotation to false and confirm that syncing is back on.

3. "apps.3scale.net/disable-apicast-staging-replica-reconciler": "true"
4. confirm that the apicast staging isn't synced anymore then set the above annotation to false and confirm that syncing is back on.

5. "apps.3scale.net/disable-backend-listener-replica-reconciler": "true"
6. confirm that the backend listeners aren't synced anymore then set the above annotation to false and confirm that syncing is back on.

7. "apps.3scale.net/disable-backend-worker-replica-reconciler": "true"
8. confirm that the backend workers aren't synced anymore then set the above annotation to false and confirm that syncing is back on.

9. "apps.3scale.net/disable-cron-replica-reconciler": "true"
10. confirm that the backend cron aren't synced anymore then set the above annotation to false and confirm that syncing is back on.

Revert updates to APIManager annotations and confirm that replicas syncing is back working.
